apps/client/apis/myStudyApi.ts

```diff
@@ -33,15 +33,14 @@ export const myStudyApi = {
 
     return response.data;
   },
-  getStudyAnnouncementList: async (studyId: number) => {
-    const response = await fetcher.get<StudyAnnouncementListDtoType>(
-      `${apiPath.basicStudyInfo}/${studyId}/${apiPath.studyAnnouncementList}`,
+  getStudyAllAnnouncementList: async () => {
```

Comment:
p4;
```suggestion
  getAllStudyAnnouncementList: async () => {
```
**모든 스터디**의 공지목록을 불러온다는 의미에 맞게 요렇게 수정해보는 건 어떨까요?